app/views/participants/validations/cannot_find_details.html.erb

```diff
@@ -0,0 +1,27 @@
+<% content_for :title, "We cannot find your details" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">We cannot find your details</h1>
+
+    <p class="govuk-body">This could be because:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>you used a different version of your first name, for example ‘Rob’ or ‘Robert’</li>
+      <li>your last name is different to the version held by the Teaching Regulation Agency</li>
+      <li>you mistyped or entered some incorrect information</li>
+    </ul>
+    
+    <p class="govuk-body">Entering your National Insurance number will help us to find your details.</p>
+
+    <%= govuk_link_to "Try again", participants_validation_tell_us_your_details_path, button: true %>
+
+    <% if @form.validation_attempts > 1 %>
+      <h2 class="govuk-heading-m">If we still cannot find your details</h2>
+      <p class="govuk-body">You can continue to register. Our support team may contact you for more information before you start your induction training.</p>
+      <%= form_for @form, url: { action: @form.step }, method: :put do |f| %>
```

Comment:
Can we please use `form_with`? https://stackoverflow.com/questions/43868976/rails-5-form-for-vs-form-with